test/JDBC/expected/securityadmin_role-vu-verify.out

```diff
@@ -0,0 +1,1520 @@
+go
+~~START~~
+int
+1
+~~END~~
+
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- psql
+SET client_min_messages = 'error';
+CREATE EXTENSION IF NOT EXISTS tds_fdw;
+GO
+
+-- tsql
+-- Add localhost as linked server
+EXEC sp_addlinkedserver  @server = N'server_4229', @srvproduct=N'', @provider=N'SQLNCLI', @datasrc=N'localhost', @catalog=N'master'
+GO
+
+EXEC sp_addlinkedserver  'Accounts'
+GO
+
+-- Add jdbc_user as linked server login
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'server_4229', @useself = 'FALSE', @rmtuser = 'jdbc_user', @rmtpassword = '12345678'
+GO
+
+drop database securityadmin_db1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+EXEC sp_addlinkedsrvlogin 'Accounts', 'False'
+GO
+
+EXEC sp_droplinkedsrvlogin 'Accounts', NULL
+GO
+
+-- inside procedure
+exec securityadmin_create_login_p1
+go
+
+exec securityadmin_alter_login_p1
+go
+
+exec securityadmin_add_mem_p1
+go
+
+exec securityadmin_drop_mem_p1
+go
+
+exec securityadmin_drop_login_p1
+go
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- login is member of both securityadmin as well as sysadmin
+drop user securityadmin_user1
+go
+
+Alter server role sysadmin add member securityadmin_login1
+go
+
+create database securityadmin_db1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+-- it should be able to connect to the database
+use securityadmin_db1
+go
+
+-- it should be dbo
+select current_user
+go
+~~START~~
+varchar
+dbo
+~~END~~
+
+
+-- both attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#1
+~~END~~
+
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- only member of securityadmin
+alter server role sysadmin drop member securityadmin_login1
+go
+
+create user securityadmin_user1 for login securityadmin_login1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+
+-- only rolcreaterole attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#0
+~~END~~
+
+
+-- should be able to create/drop login
+create login test_securityadmin_l1 with password ='123'
+go
+
+drop login test_securityadmin_l1
+go
+
+-- terminate-tsql-conn user=securityadmin_login1 password=123
+
+-- tsql
+-- only member of sysadmin
+drop user securityadmin_user1
+go
+
+alter server role sysadmin add member securityadmin_login1
+go
+
+alter server role securityadmin drop member securityadmin_login1
+go
+
+-- terminate-tsql-conn
+
+-- tsql user=securityadmin_login1 password=123
+
+-- both attribute should be true
+select rolname, rolcreaterole, rolcreatedb from pg_roles where rolname = 'securityadmin_login1'
+go
+~~START~~
+varchar#!#bit#!#bit
+securityadmin_login1#!#1#!#1
+~~END~~
```

Comment:
can we test the `is_srvrolemember` for this case as well where it is member of sysadmin but not securityadmin? If we haven't add already at anywhere else.